ucm2/AMD/acp5x/acp5x.conf

```diff
@@ -0,0 +1,8 @@
+Syntax 3
```

Comment:
You should use DMI or any other information to use this acp5x configuration only for this specific hardware.

Reply:
Thanks for reviewing!

I have just pushed an updated version, restricting the usage to the Steam Deck hardware.